package.xml

```diff
@@ -17,6 +17,7 @@
   <author>Ken Conley</author>
 
   <buildtool_depend version_gte="0.5.74">catkin</buildtool_depend>
+  <run_depend version_gte="0.5.74">catkin</run_depend>
```

Comment:
I don't think there is a reason for specifying a minimum version of catkin. All the functions and variables used at runtime exist in catkin "forever".